docs: update plugin documentation (#888)
jackyzha0/quartz: docs/plugins/Assets.md

---
title: Assets
tags:
  - plugin/emitter
---

> This plugin manages and emits non-Markdown assets (like images, scripts, stylesheets, etc.) from the content directory to the output directory. It ensures that all required static assets are available in the generated site.

Reviewer: ```suggestion
This plugin emits all non-Markdown static assets (like images, scripts, stylesheets, etc.) from your vault content under the directory `content`.

Note that all static assets will then be accessible through its path on your generated site, i.e: `host.me/path/to/static.pdf`
```

---
title: Assets
tags:
  - plugin/emitter
---

> This plugin emits all non-Markdown static assets (like images, scripts, stylesheets, etc.) from your vault content under the directory `content`.

Reviewer 2: ```suggestion
This plugin emits all non-Markdown static assets in your content folder (like images, videos, HTML, etc).
```

> Note that all static assets will then be accessible through its path on your generated site, i.e: `host.me/path/to/static.pdf`

Reviewer 2: also note that this respects `ignorePatterns`

Author: Done
